src/Serilog/Core/Logger.cs

```diff
@@ -1328,7 +1327,7 @@ public void Fatal(Exception? exception, string messageTemplate, params object?[]
         /// }
         /// </example>
         [MessageTemplateFormatMethod("messageTemplate")]
-        public bool BindMessageTemplate(string messageTemplate, object?[]? propertyValues,
+        public bool BindMessageTemplate(string? messageTemplate, object?[]? propertyValues,
```

Comment:
I don't think the intention is for `messageTemplate` to be nullable here; IIRC the dynamic check against null is just to preserve the "no throw" guarantee (as for the same method on `ILogger`).

Reply:
fixed